environment/src/main/resources/schema/app/20210302145523_CB-11152:_Add_keyUrl_CLI_param_in_Azure_environment_creation.sql

```diff
@@ -0,0 +1,7 @@
+-- // CB-11152: Add keyUrl CLI param in Azure environment creation
+-- Migration SQL that makes the change goes here.
+ALTER TABLE environment_parameters ADD COLUMN IF NOT EXISTS encryption_parameter_keyUrl varchar(255);
+
+-- //@UNDO
+-- SQL to undo the change goes here.
+ALTER TABLE environment_parameters DROP COLUMN IF EXISTS encryption_parameter_keyUrl;
```

Comment:
With the above entity change, `encryption_parameter_keyUrl` shall be replaced with `encryption_keyUrl`. Be sure to first roll back the original SQL script in your local DB before renaming the column here.